deploy/helm/sumologic/templates/pvc-cleaner/cron-job.yaml

```diff
@@ -0,0 +1,121 @@
+{{- if eq (dig "pvc-cleaner" "logs" "enabled" false (.Values | merge (dict))) true }}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ template "sumologic.metadata.name.pvc-cleaner.logs" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ template "sumologic.labels.app.pvc-cleaner.logs" . }}
+    {{- include "sumologic.labels.common" . | nindent 4 }}
+spec:
+  schedule: "*/15 * * * *"
+  jobTemplate:
```

Comment:
The schedule should probably be configurable, this would be especially useful for potential integration tests.